packages/rich-text/src/component/index.js

```diff
@@ -392,8 +386,17 @@ class RichText extends Component {
 
 	/**
 	 * Handles the `selectionchange` event: sync the selection to local state.
+	 *
+	 * @param {Event} event `selectionchange` event.
 	 */
-	onSelectionChange() {
+	onSelectionChange( event ) {
```

Comment:
@gziolo Is there any way you could try the changes to this file with the new puppeteer? Or how could I try it? I have a feeling this might work.